source/server/server.cc

```diff
@@ -530,7 +530,7 @@ void InstanceImpl::initialize(Network::Address::InstanceConstSharedPtr local_add
                                      : ReusePortDefault::False;
   } else {
     // This is needed so that we don't read the value until runtime is fully initialized.
-    enable_reuse_port_default_ = ReusePortDefault::Runtime;
+    enable_reuse_port_default_ = ReusePortDefault::True;
```

Comment:
from the 1.20.oc doc, the default value is set to false, please correct me if I'm wrong.
ref to v1.20.0.rst "```
To retain the previous behavior, either explicitly set the new configuration
  field to false, or set the runtime feature flag ``envoy.reloadable_features.listener_reuse_port_default_enabled``
  to false.
```"

Reply:
I think the default value is true from my experience, and this feature is enabled by default.

Reply:
Correct, the new default is true.